test/facility-fixed-fee/post-facility-fixed-fees-amount-amendment.api-test.ts

```diff
@@ -0,0 +1,238 @@
+    expect(body).toStrictEqual({
+      bundleIdentifier: createdBundleIdentifier,
+    });
+    expect(acbsRequest.isDone()).toBe(true);
+  });
+
+  it('returns the bundleIdentifier of the created bundle in ACBS if creating a decrease fixed fees amount amendment is successful', async () => {
+    givenAuthenticationWithTheIdpSucceeds();
+    const acbsRequest = requestToCreateDecreaseFixedFeesAdvanceTransactionInAcbs().reply(...acbsSuccessfulResponse);
+
+    const { status, body } = await api.post(createFixedFeesAmountAmendmentUrl(), decreaseAmountRequest);
+
+    expect(status).toBe(201);
+    expect(body).toStrictEqual({
+      bundleIdentifier: createdBundleIdentifier,
+    });
+    expect(acbsRequest.isDone()).toBe(true);
+  });
+
+  describe('error cases when creating the fixed fees advance transaction', () => {
+    beforeEach(() => {
+      givenAuthenticationWithTheIdpSucceeds();
+    });
+
+    it('returns a 404 response if ACBS responds with a 400 response that is a string containing "Facility does not exist" when creating the facility fixed fees amount amendment', async () => {
+      requestToCreateIncreaseFixedFeesAdvanceTransactionInAcbs().reply(
+        400,
+        `Facility does not exist or user does not have access to it: '${facilityIdentifier}'`,
+      );
+
+      const { status, body } = await api.post(createFixedFeesAmountAmendmentUrl(), increaseAmountRequest);
+
+      expect(status).toBe(404);
+      expect(body).toStrictEqual({ message: 'Not found', statusCode: 404 });
+    });
+
+    it('returns a 400 response if ACBS responds with a 400 response that is not a string when creating the facility fixed fees amount amendment', async () => {
+      const acbsErrorMessage = JSON.stringify({ Message: 'error message' });
+      requestToCreateIncreaseFixedFeesAdvanceTransactionInAcbs().reply(400, acbsErrorMessage);
+
+      const { status, body } = await api.post(createFixedFeesAmountAmendmentUrl(), increaseAmountRequest);
+
+      expect(status).toBe(400);
+      expect(body).toStrictEqual({ message: 'Bad request', error: acbsErrorMessage, statusCode: 400 });
+    });
+
+    it('returns a 400 response if ACBS responds with a 400 response that is a string that does not contain "Facility does not exist" when creating the facility fixed fees amount amendment', async () => {
+      const acbsErrorMessage = 'ACBS error message';
+      requestToCreateIncreaseFixedFeesAdvanceTransactionInAcbs().reply(400, acbsErrorMessage);
+
+      const { status, body } = await api.post(createFixedFeesAmountAmendmentUrl(), increaseAmountRequest);
+
+      expect(status).toBe(400);
+      expect(body).toStrictEqual({ message: 'Bad request', error: acbsErrorMessage, statusCode: 400 });
+    });
+
+    it('returns a 500 response if ACBS responds with an error code that is not 400 when creating the facility fixed fees amount amendment', async () => {
+      requestToCreateIncreaseFixedFeesAdvanceTransactionInAcbs().reply(401, 'Unauthorized');
```

Comment:
* Will amount `0` be covered in this assertion?
* Will a invalid segment identifier also be covered in this section (albeit an edge case).

Reply:
Not this specific test, but field input is checked in this file.

It is in this describe section
```
describe('field validation', () => {
```